psydac/feec/global_projectors.py

```diff
@@ -29,11 +29,11 @@ def __init__(self, H1):
         self.args  = (*n_basis, *points, self.rhs._data[slices])
 
```

Comment:
Projector docstrings should probably indicate whether they are to be applied to functions (forms) in the logical or physical domain

Reply:
@campospinto I've updated the docstrings of the H1 projector in commit 6457ee9. Please let me know what you think of it!

I will then proceed with updating the docstrings of the other projectors.

Reply:
Very clear now, thanks

Reply:
I updated the docstrings of the other projectors in commit aceccae.